qa/workunits/cephadm/test_iscsi_pids_limit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+# checks if the containers default pids-limit (4096) is removed and Iscsi
+# containers continue to run
+# exits 1 if fails
+
+# checks which container engine is installed
+if which podman 2>&1 >/dev/null; then
+  CONT_CMD="podman"
+else
+  CONT_CMD="docker"
+fi
+
+ISCSI_CONT_IDS=$(${CONT_CMD} ps -qa --filter='name=iscsi')
+
+for i in $ISCSI_CONT_IDS
```

Comment:
Nit: you are using `${VAR}` syntax in other places even though it's not needed.  It would be nice to be consistent: either stick to braces everywhere or don't use them at all.